packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-tooltip.tsx

```diff
@@ -0,0 +1,100 @@
+import { Component, Element, h, Prop } from '@stencil/core';
+
+import tippy, { Instance } from 'tippy.js';
+import { DataPoint } from '../../../utils/dataTypes';
+import { Threshold } from '../../charts/common/types';
+import { TIPPY_SETTINGS } from '../../common/toolTipSettings';
+import { Value } from '../../value/Value';
+
+@Component({
+  tag: 'sc-dial-tooltip',
+  styleUrl: 'sc-dial-base.css',
+  shadow: false,
+})
+export class ScDialTooltip {
```

Comment:
is there some way not to duplicate `sc-grid-tooltip` but instead refactor and se a shared component?

Reply:
Could I directly change the `SC-grid-ToolTip` component to a shared component？Add some of the things Dial needs to the `SC-grid-ToolTip` component？

Reply:
yes it may take a little refactoring but I think that's the route to go

Reply:
Done --- I modify `sc-grid-tooltip` props.